src/victory-label/victory-label.js

```diff
@@ -264,10 +267,14 @@ export default class VictoryLabel extends React.Component {
           const style = this.style[i] || this.style[0];
           const lastStyle = this.style[i - 1] || this.style[0];
           const fontSize = (style.fontSize + lastStyle.fontSize) / 2;
+          const lineHeight = Array.isArray(this.lineHeight)
```

Comment:
Do we need to detect/handle empty arrays? (Same question for existing `style` var too)

Reply:
I suppose for completeness sake we should, but in both of those cases, that would only happen if a user _explicitly_ added the prop like `lineHeight={[]}`. That would technically be allowed by the prop definition, but doesn't make any intuitive sense. I think it's a highly unlikely edge case to hit, but I agree that it's bad practice to trust the user to not break things.

Reply:
@parkerziegler can you add a fallback for empty arrays for `lineHeight` in this PR. I'll create a new issue for styles

Reply:
@boygirl @ryan-roemer Sure thing, I'll do it first thing when I get in tomorrow.

Reply:
@ryan-roemer I think I'm pretty much all set to push the changes. We fallback to default values for both `style` and `lineHeight` if empty arrays are passed for those props. Should I add a `console.warn` just to let users know this is **not** a recommended practice?